### PR TITLE
Add debugging page to sidebar

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
This page was not in the sidebar

# Reasons
Sidebar reorg

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/high-uid-error.md [hunk 1]
```diff
@@ -64,46 +61,39 @@ you must update the files' UID/GID
 and re-create the image.
 
 If you are not the image maintainer, congratulations:
```
Comment (kbatuigas) [inline jekyll/_cci2/high-uid-error.md:63]: > If you are not the image maintainer, congratulations:

I get the sentiment but not sure it's necessary in our docs 😆

**Review by kbatuigas — APPROVED** [review]
One nitpick but approved :)

File: jekyll/_cci2/high-uid-error.md (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/high-uid-error.md:63]: Yeah I dithered about that one but you are right!

Comment (rosieyohannan) [inline jekyll/_cci2/high-uid-error.md:63]:
```suggestion
If you are not the image maintainer, 
```

**Pull request merged**